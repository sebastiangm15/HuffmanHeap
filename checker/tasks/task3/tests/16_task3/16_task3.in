13
2 K1
3 K2
4 K3
5 K4
6 K5
7 K6
8 K7
9 K8
10 K9
11 K10
12 K11
13 K12
14 K13
5
K1
K5
K7
K9
K13
