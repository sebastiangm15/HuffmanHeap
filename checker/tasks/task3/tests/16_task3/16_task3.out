11101001011101000101
